diabetes_regression/evaluate/evaluate_model.py

```diff
@@ -26,7 +26,11 @@
 from azureml.core import Run
 import argparse
 import traceback
-from util.model_helper import get_model
+from diabetes_regression.util.model_helper import get_model
```

Comment:
As @j-so mentioned, all the references to `diabetes_regression` by name will not work when someone bootstraps and renames the project. We should find a way to reference this using a variable.  Both in the python files and in the pipelines.